client/cards/next.js

```diff
@@ -18,6 +13,11 @@ module.exports = h('.next',
   */
 
   h('div', {style: {'margin-top': '10%'}},
+    h('h2', 'Justin Dorfman'),
+    h('h2', 'Let’s talk about SRI')
+  ),
```

Comment:
I believe @jcblw said we have a second speaker lined up but I don't see it on the spread sheet.

@jcblw can we get this info to put up on the slides tonight?